maestrowf/datastructures/core/parameters.py

```diff
@@ -299,8 +299,11 @@ def get_combinations(self):
             combo = Combination()
             for key in self.parameters.keys():
                 pvalue = self.parameters[key][i]
-                tlabel = self.labels[key].replace(self.label_token,
-                                                  str(pvalue))
+                try:
```

Comment:
I still don't like this `try/catch` method because we could just as easily check to see if something is a list. My suggestion is to use an `if/else` with the `isinstance(x, type)` in this case.

Reply:
@FrankD412 I was trying to avoid the py2/3 basestr,str issue. I could check for a list and assume it's a string otherwise.

Reply:
@koning -- I've been assuming `str` which puts us in the realm of ASCII. For now, assume we'll be seeing `str`. It'll be a separate project to go catch `unicode` later.